docs/compute-element/install-htcondor-ce.md

```diff
@@ -7,7 +7,7 @@ The [HTCondor-CE](http://htcondor-ce.org) software is a *job gateway* for an OSG
 As such, the OSG will submit resource allocation requests (RARs) jobs to your HTCondor-CE and it will handle
 authorization and delegation of RARs to your local batch system.
 In OSG today, RARs are sent to CEs as *pilot jobs* from a factory, which in turn are able to accept and run end-user jobs.
-See the [upstream documentation](https://htcondor.github.io/htcondor-ce/architecture/) for a more detailed introduction.
+See the [upstream documentation](https://htcondor.com/htcondor-ce/architecture/) for a more detailed introduction.
```

Comment:
Maybe you should leave this as htcondor.github.io until htcondor.org gets fixed...

Reply:
htcondor.com works, though, and Todd owns that!